src/app/modules/form/admin-form/admin-form.assistance.controller.ts

```diff
@@ -81,3 +107,99 @@ export const handleTextPrompt = [
   handleTextPromptValidator,
   _handleTextPrompt,
 ] as ControllerHandler[]
+
+const handleVisionPromptValidator = celebrate({
+  [Segments.PARAMS]: {
+    formId: Joi.string()
+      .required()
+      .pattern(/^[a-fA-F0-9]{24}$/)
+      .message('Your form ID is invalid.'),
+  },
+  [Segments.BODY]: {
+    imageDataUrls: Joi.array().items(Joi.string()).required().min(1).max(10),
```

Comment:
Recommending for us to move the `10` in `max(10)` to a shared constant since frontend(`frontend/src/features/admin-form/create/builder-and-design/MagicFormBuilder/components/MagicFormBuilderPromptModal.tsx`) shares it

Reply:
moved!